.github/workflows/code-health.yml

```diff
@@ -10,17 +10,17 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+      - uses: GitHubSecurityLab/actions-permissions/monitor@e83639073897b68de235d81a783b2221cc13def7
         with:
           config: ${{ vars.PERMISSIONS_CONFIG }}
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
```

Comment:
my understanding was that actions supported by GH could remain on numbers and not shas, has this changed?

Reply:
using shas is actually really bad for security scanning tools as you may be able to  see from https://github.com/mongodb/mongodb-atlas-cli/pull/3063

Reply:
I was unaware of this, could you share where it was decided?

Reply:
no problem if you don't find it. It makes sense that GH actions can be trusted